src/vitabel/vitals.py

```diff
@@ -1843,6 +1843,10 @@ def find_CC_periods_acc(
         if isinstance(accelerometer_channel, str):
             if accelerometer_channel in self.get_channel_names():
                 ACC_channel = self.get_channel(accelerometer_channel)
+            elif accelerometer_channel == 'cpr_acceleration':
+                logger.error(
+                "WARNING! The channel 'cpr_acceleration' (default) does not exist in the recording. "
+                "Please specify a channel or a string with the name of the channel.")
```

Comment:
Should this really be a separate logger error, or should we just make the generic `ValueError` thrown below more helpful?